PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -579,14 +585,60 @@ def miniAOD_customizeData(process):
     ctpps_2016.toModify(task, func=lambda t: t.add(process.ctppsLocalTrackLiteProducer))
     ctpps_2016.toModify(task, func=lambda t: t.add(process.ctppsProtons))
 
+def miniAOD_customizeHeavyIon(process, data):
+    from PhysicsTools.PatAlgos.producersHeavyIons.heavyIonJetSetup import aliasCsJets, removeL1FastJetJECs, removeJECsForMC, addJECsForData
+    from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+    from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+    (pp_on_AA_2018 | pp_on_PbPb_run3).toModify(process.slimmedJets, src = 'selectedPatJets')
```

Comment:
I still think that all .toModify of already existing common modules should be done either in the file where the module is defined or if it's inserted via a tool in the place where the tool is called; here many of the updates will need to be moved to ```miniAOD_customizeCommon```.

Perhaps a more palatable solution with a more straightforward syntax is 
```python
miniAOD_customizeHeavyIonModifier = (pp_on_AA_2018 | pp_on_PbPb_run3).makeProcessModifier(miniAOD_customizeHeavyIon)
```
to keep miniAOD_tools size under control, perhaps define this modifier in a separate file and just load it at the end of `miniAOD_customizeCommon`
This way, inside of the modification method you do not need to repeat `(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(`

define miniAOD_customizeHeavyIon

Reply:
I wasn't easily able to get this to work.  Can you make this suggestion a bit more verbose?

Reply:
Here is what I meant

- make a file miniAOD_customizeHeavyIon.py
- ```def miniAOD_customizeHeavyIon(process):``` and all contents of the current miniAOD_customizeHeavyIon without ```(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(```
- after the definition add ```miniAOD_customizeHeavyIonModifier = (pp_on_AA_2018 | pp_on_PbPb_run3).makeProcessModifier(miniAOD_customizeHeavyIon)``` at the end of the file
- remove miniAOD_customizeHeavyIon from miniAOD_tools.py 
- add the following line at the end of ```miniAOD_customizeCommon```: ```from PhysicsTools.PatAlgos.slimming import miniAOD_customizeHeavyIonModifier```

this should deliver the same result with a shorter syntax
(disclaimer: I didn't put much thought into data vs MC cases)

Reply:
Ok, that's basically what I had tried, but I'm finding that the customization are not being implemented.
My latest attempt may be found here:
/afs/cern.ch/work/m/mnguyen/public/integration/CMSSW_11_2_X_2020-08-24-1100/src/PhysicsTools/PatAlgos/python/slimming/
miniAOD_tools.py
miniAOD_customizeHeavyIon